src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Windows.props

```diff
@@ -14,4 +14,9 @@ Copyright (c) .NET Foundation. All rights reserved.
   <ItemGroup Condition=" '$(_IncludeWindowsSDKRefFrameworkReferences)' == 'true' ">
     <FrameworkReference Include="Microsoft.Windows.SDK.NET.Ref" IsImplicitlyDefined="true" Pack="false" PrivateAssets="All" />
   </ItemGroup>
+
+  <!--https://github.com/dotnet/sdk/issues/12403-->
+  <PropertyGroup>
+    <_TargetFrameworkVersionValue>0.0</_TargetFrameworkVersionValue>
```

Comment:
@dsplaisted I added this to workaround the missing property issue while I am checking into dotnet/wpf